client/reader/search-stream/suggestion-provider.jsx

```diff
@@ -11,10 +11,11 @@ import { map, sampleSize } from 'lodash';
 import { getLocaleSlug } from 'lib/i18n-utils';
 import { suggestions } from 'reader/search-stream/suggestions';
 import getReaderFollowedTags from 'state/selectors/get-reader-followed-tags';
-import analytics from 'lib/analytics';
+import { createRandomId } from '@automattic/calypso-analytics';
```

Comment:
I'm not sure if it makes sense to export `createRandomId` from the analytics library. It produces a general-purpose random string. Not specific to analytics in any way.

In this Reader file, it seems to be used for a non-analytics purpose. Or is the railcars code intended for reporting to analytics? @blowery might know more 🙂

Reply:
ok, I couldn't see a good place to move it to and I think it is a bit small to justify its own package so I've added a new package `calypso-utils` with just createRandomId in it for now, does that sound good to you?

Reply:
I think that creating a new package is not worth it, especially if the package doesn't have a clear purpose and is just a bag of random utilities.

To be honest, I would just duplicate the code at this moment 🙂 Or find an existing NPM package that generates a random base64 string. `randomstring` or `crypto-random-string` look good.

Reply:
ahh true, ok I'll look into one of those packages

Reply:
I had a look but I couldn't get the packages working in the browser (they use node crypto), I just went with duplicating the code between those two places it was used

Reply:
> I couldn't get the packages working in the browser 

Oh, that's a bummer that the popular packages are Node.js only 😢 It seems that, after all, there is still a market for a tiny `@automattic/random-string` package 🙂